### ISSUE TITLE
local modules are evaluated more than once

### ISSUE THREAD

**Issue by bzb-stcnx** [ChatTriggers/ChatTriggers#379]
!!! congrats for this awesome mod :-) !!!

**Describe the bug**
Modules from the local `modules` folder are evaluated a first time when automatically imported at start-up, and a second time when imported from another module.

**To Reproduce**
1. create two simple modules in the local `modules` directory:
```
modules
├── test1
│   ├── index.js
│   └── metadata.json
└── test2
    ├── index.js
    └── metadata.json
```
`modules/test1/metadata.json`:
```json
{
  "name": "test1",
  "creator": "bzb-stcnx",
  "version": "1.0.0",
  "entry": "index.js",
  "requires": []
}
```
`modules/test1/index.js`:
```js
console.log('###', JSON.stringify(module))
```
`modules/test1/metadata.json`:
```json
{
  "name": "test2",
  "creator": "bzb-stcnx",
  "version": "1.0.0",
  "entry": "index.js",
  "requires": ["test1"]
}
```
`modules/test1/index.js`:
```js
import "test1"
console.log('###', JSON.stringify(module))
```
2. launch MC, login to hypixel and open the js console:
```
/ct console js
```
3. sample console output:
```
### {"id":"test1","uri":"file:/home/bzb/.minecraft/config/ChatTriggers/modules/test1/index.js","exports":{}}
### {"id":"test2/index.js","uri":"file:/home/bzb/.minecraft/./config/ChatTriggers/modules/test2/index.js","exports":{}}
### {"id":"test1/index.js","uri":"file:/home/bzb/.minecraft/./config/ChatTriggers/modules/test1/index.js","exports":{}}
```
the `test1` module code is evaluated twice, i.e. the `console.log` statement is run twice.

**Expected behavior**
ECMAScript spec mandates the module code is only evaluated once when imported multiple times.
In fact, the CT code is built accordingly: it caches module upon first import.
Basically, we would expect to only see the first `console.log` output from the `test1` module, not the second.

However, here both imports of the `test1` module are considered to be requiring different modules.
Indeed both their `module.id` and `module.uri` are *different strings*, even though `module.uri` should ultimately be recognized as the *same file path*.

**Setup:**
 - OS: ArtixLinux
 - ChatTriggers Version: 2.2.0
 - Minecraft Version: 1.8.9
- Java Version: openjdk 20.0.1

**Comment by bzb-stcnx** [comment]
a workaround seems to be to use the full module reference `test1/index.js` both in the import statement in `test2/index.js` and in the `requires` array in `test2/metadata`, in which case the `module.id` is the same and both imports are considered requiring the same module.
in fact, since the automatic import already refers to `test1/index.js`, only the import statement in `test2/index.js` needs to be updated to the explicit filepath: `import 'test1/index.js'`.

**Comment by mattco98** [comment]
Thanks for the detailed bug report! I've known about this issue for a while, but never the cause (i.e. different paths to the same module). This should be fairly simple to fix, just normalize the paths before loading them.
